auto_nag/tests/test_query_creator.py

```diff
@@ -16,8 +14,8 @@ def test_1_getTemplateValue(self):
         Tests for getTemplateValue,
         Expecting a VERSION number
         """
-        url = "https://wiki.mozilla.org/Template:BETA_VERSION"
-        beta_version = getTemplateValue(url)
```

Comment:
this is a test for getTemplateValue, we shouldn't silently make it test something else (and we should keep a test for getTemplateValue as long as that exists and is used for anything)